addons/binding/org.openhab.binding.astro/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+  <config-description uri="config:astro-config">
+	<parameter name="latitude" type="decimal" required="true">
```

Comment:
I would suggest merging latitude and longitude into one string property (syntax "51.343,-140.32") with context "geolocation". This would allow UIs to use a map to directly pick a location. Having these both parameters separately makes it more complicated.